pkg/cmd/openshift-tests/health-check/command.go

```diff
@@ -0,0 +1,31 @@
+package health_check
```

Comment:
can we change this to `post_analysis` that way we can build this out with future work that may not be specific to cluster health.  Same for any `HealthCheck` references, consider `PostAnalysis` for the command specific code.